docs/content/en/docs/Advanced/coco.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Confidential computing setup"
+linkTitle: "Confidential Computing"
+weight: 9
+description: >
+---
+
+Confidential computing is a type of secure computing that allows users to encrypt and decrypt data on a secure, isolated computing environment.
+It works by encrypting the data before it is sent to the cloud or other computing resources. This allows users to keep their data private and secure, even if it is accessed by unauthorized parties.
+This makes it useful for sensitive data such as financial information, health records, and other confidential data.
+
+One important aspect of Confidential Computing is the ability to encrypt data even in-memory. This document describes how to setup Kairos to use [`enclave-cc`](https://github.com/confidential-containers/enclave-cc)
+in order to run confidential workloads.
+
+## Create a Kairos cluster
+
+The [`coco community bundle`](https://github.com/kairos-io/community-bundles/tree/main/coco) is supported since Kairos version `v2.0.0-alpha3` ("coco" stands for "**Co**nfidential **Co**mputing).
```

Comment:
```suggestion
The [`coco community bundle`](https://github.com/kairos-io/community-bundles/tree/main/coco) is supported since Kairos version `v2.0.0-alpha3` ("coco" stands for "**Co**nfidential **Co**mputing").
```